src/object/srv_obj.c

```diff
@@ -3004,6 +3033,9 @@ ds_obj_rw_handler(crt_rpc_t *rpc)
 			       DP_DTI(&orw->orw_dti), DP_RC(rc1));
 	}
 
+	if (ioc.ioc_map_ver < max_ver)
+		ioc.ioc_map_ver = max_ver;
```

Comment:
is it the case the client IO RPC's map_ver is larger than server-side pool's map ver right? why reset ioc.ioc_map_ver here?

Reply:
It is that some DTX participant (such as some replica) has large pool map version than client, then reply it back to client.